File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int ind=0;
        int n=nums.length;

        for(int i=0;i<n;i++){
            if(i>0 && nums[i]==nums[i-1]) continue;
            nums[ind]=nums[i];
            ind++;
        }
        return ind;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let ind=0;

    for(let i=1;i<nums.length;i++){
        if(nums[ind]!=nums[i]){
            ind++;
            nums[ind]=nums[i];
        }
    }
    return ind+1;
};